### ISSUE TITLE
File Operations on selected files

### ISSUE THREAD

**Issue by pepsakdoek** [sla80/SmartSvgViewer#3]
I'm loving that I can view a ton of SVG files at once, and select them, but currently I can't do anything with them. So delete/copy/move/mass rename might all be very useful features to have on a folder full of images.

**Comment by sla80** [comment]
@pepsakdoek Yes, the application already has the ability to select multiple files, and later a menu with some standard file actions will be added.

**Comment by sla80** [comment]
Hello @pepsakdoek!

Update 0.5.0.1685 has been released. Now, if you select multiple (!) images in the gallery (using Ctrl+Click or Shift+Click), and then Right-Click on any of them, then a Windows Explorer will open, with all these multiple SVG files selected. So you can perform actions on multiple files in Explorer - Copy, Cut, Delete. etc.

Also, maybe later it will be possible to do file operations in the application itself, but turning an application into a file manager is a more difficult task than it seems.
